.gitignore

```diff
@@ -13,6 +13,7 @@ __pycache__/
 
 # C extensions
 *.so
+cytrics_venv
```

Comment:
nit - I'd probably add this to the `Environments` section that begins around line 131 of this file.